stan/math/opencl/device_functions/log1m_exp.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef STAN_MATH_OPENCL_DEVICE_FUNCTIONS_LOG1M_EXP_HPP
+#define STAN_MATH_OPENCL_DEVICE_FUNCTIONS_LOG1M_EXP_HPP
```

Comment:
I missed this one on the digamma case, but can we move the device functions inside kernels. They are only going to be used inside kernels, so I think that makes most sense.

Reply:
I am also moving digamma in this PR.

Reply:
Yes, please do.